src/test_utils.cairo

```diff
@@ -0,0 +1,46 @@
+//! Test helpers
+
+/// Get bytes from hex (base16)
+pub fn from_hex(hex_string: ByteArray) -> ByteArray {
+    let num_characters = hex_string.len();
+    assert!(num_characters & 1 == 0, "Invalid hex string length");
```

Comment:
just for info but i suppose you already know that
`assert!` macro involves like 10 times more steps than `assert` function.
If there is no requirement to add a very long string bigger than a felt252 for the error message, its way more efficient to use `assert` function than the macro. 

```
assert(num_characters & 1 == 0, 'Invalid hex string length');
```

but anyway this is for tests only, so i guess it doesnt matter that much